test/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoaderTest.java

```diff
@@ -3674,5 +3674,34 @@ private IInternalContest fullLoadSampleContest(String sampleName) throws Excepti
         return contest;
     }
     
+    
+    /**
+     * Test halt-contest-clock-at-set to true.
+     * 
+     * @throws Exception
+     */
+    public void testisHaltContestAtTimeZero() throws Exception {
+        String sampleContestDirName = "ccs1";
+        String dirname = getContestSampleCDPConfigDirname(sampleContestDirName);
+        
+        IInternalContest contest = snake.fromYaml(null, dirname, false);
+        assertNotNull("Expecting to load ccs1 contest",contest);
+        assertTrue("Expected halt at end of contest ", contest.getContestInformation().isAutoStartContest());
+    }
+    
+    /** 
+     * Test halt-contest-clock-at-end value, for when missing key/value
+     * @throws Exception
+     */
+    public void testisHaltContestAtTimeZeroNegative() throws Exception {
+        String sampleContestDirName = "ccs2";
+        String dirname = getContestSampleCDPConfigDirname(sampleContestDirName);
+        
+        IInternalContest contest = snake.fromYaml(null, dirname, false);
+        assertNotNull("Expecting to load ccs2 contest",contest);
+        assertFalse("Expected NO halt at end of contest ", contest.getContestInformation().isAutoStartContest());
```

Comment:
Same issue as above:  it appears to me that this is calling the wrong method.

Reply:
same typo - fixing.